packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -146,127 +148,63 @@ export class GutenbergEditorPage {
 	}
 
 	/**
-	 * Enters the text into the title block and verifies the result.
+	 * Closes all panels that can be opened in the editor.
 	 *
-	 * @param {string} title Text to be used as the title.
-	 * @returns {Promise<void>} No return value.
-	 * @throws {assert.AssertionError} If text entered and text read back do not match.
+	 * This method will attempt to close the following panels:
+	 * 	- Publish Panel (including pre-publish checklist)
+	 * 	- Editor Settings Panel
+	 * 	- Editor Navigation Sidebar
 	 */
-	async enterTitle( title: string ): Promise< void > {
-		const sanitizedTitle = title.trim();
-		await this.setTitle( sanitizedTitle );
-		const readBack = await this.getTitle();
-		assert.strictEqual( readBack, sanitizedTitle );
+	async closeAllPanels(): Promise< void > {
+		await Promise.allSettled( [
+			this.editorPublishPanelComponent.closePanel(),
+			this.editorNavSidebarComponent.closeSidebar(),
+			this.editorToolbarComponent.closeSettings(),
+		] );
 	}
 
+	/* Editor */
+
 	/**
-	 * Fills the title block with text.
+	 * Enters the text into the title block and verifies the result.
 	 *
 	 * @param {string} title Text to be used as the title.
-	 * @returns {Promise<void>} No return value.
+	 * @throws {Error} If entered title does not match.
 	 */
-	async setTitle( title: string ): Promise< void > {
-		const frame = await this.getEditorFrame();
-		await frame.click( selectors.editorTitle );
-		await frame.fill( selectors.editorTitle, title );
-	}
+	async enterTitle( title: string ): Promise< void > {
+		const sanitizedTitle = title.trim();
```

Comment:
Should we move this down into the `EditorGutenbergComponent`? I feel like we want to trim all text going in there? I'm viewing this from the perspective of "let's have all the logic in one place as much as possible"

Reply:
Sure, changed to pass the original text to the method in `EditorGutenbergComponent`. However, since we're checking the work in subsequent steps, the call to sanitize the string remains in the calling method in `EditorPage`, just shifted a bit below.